verticapy/sql/flex.py

```diff
@@ -47,6 +47,40 @@ def compute_flextable_keys(
     List of tuples
         List  of virtual column names and  their
         respective data types.
+
+    Examples
+    --------
+    Create and injest a JSON file, then compute its flextable keys and predicted data types:
+
+    .. ipython:: python
+        :okwarning:
+
+        import verticapy as vp
+        from verticapy.sql import compute_flextable_keys
+        import json
+
+        data = {
+            "column1": {
+                "subcolumn1A": "value1A",
+                "subcolumn1B": "value1B"
+                },
+            "column2": {
+                "subcolumn2A": "value2A",
+                "subcolumn2B": "value2B"
+                }
+            }
+
+        json_string = json.dumps(data, indent=4)
+
+        with open("nested_columns.json", "w") as json_file:
+            json_file.write(str(json_string))
+
+        vp.create_schema("temp")
+        @suppress
+        vp.drop("temp.test")
+        vdf = vp.read_json("nested_columns.json", schema = "temp", table_name = "test", flatten_maps = False, materialize = False)
+        compute_flextable_keys(flex_name = "temp.test")
```

Comment:
Do you display the result? I have the impression that you suppress the block which includes the function. Besides, it would be better if you separate the option into two blocks: initialisation and function usage.

Reply:
output is displayed